### DISCUSSION TITLE
Sentry configuration with appsettings.json and use EntityFramework Core

### DISCUSSION THREAD

**Discussion by wingi11** [getsentry/sentry-dotnet#2069] [Q&A] [answered]
Is it possible in sentry to use the EntityFramework Module and have the rest of the configuration in the appsettings.json file?

**Comment by mattjohnsonpint** [comment, marked as answer]
Yes.  There's nothing special you need to do for this, as long as you have `appsettings.json` wired up correctly through .NET's configuration providers.

Also, I notice in the title you mention Entity Framework Core.  That's baked in and on by default in most cases.  You don't need to add anything.

The `Sentry.EntityFramework` module is for older EF 6, not EF Core.
